aries_cloudagent/revocation/manager.py

```diff
@@ -294,6 +295,9 @@ async def publish_pending_revocations(
                         )
                     else:
                         rev_entry_resp = await issuer_rr_upd.send_entry(self._profile)
```

Comment:
Before I removed this it had a `if not connection_id:` statement on this. I think that's partly what confused me initially before I got more familiar with the flow.